ksql-serde/src/main/java/io/confluent/ksql/serde/avro/KsqlGenericRowAvroDeserializer.java

```diff
@@ -16,62 +16,67 @@
 
 package io.confluent.ksql.serde.avro;
 
+import io.confluent.kafka.schemaregistry.client.SchemaRegistryClient;
+import io.confluent.kafka.serializers.KafkaAvroDeserializer;
 import io.confluent.ksql.GenericRow;
 import io.confluent.ksql.util.KsqlException;
-import org.apache.avro.Schema;
+import io.confluent.ksql.util.SchemaUtil;
+
 import org.apache.avro.generic.GenericData;
-import org.apache.avro.generic.GenericDatumReader;
 import org.apache.avro.generic.GenericRecord;
-import org.apache.avro.io.Decoder;
-import org.apache.avro.io.DecoderFactory;
 import org.apache.kafka.common.errors.SerializationException;
 import org.apache.kafka.common.serialization.Deserializer;
+import org.apache.kafka.connect.data.Field;
+import org.apache.kafka.connect.data.Schema;
 
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 
 public class KsqlGenericRowAvroDeserializer implements Deserializer<GenericRow> {
 
-  private final org.apache.kafka.connect.data.Schema schema;
+  private final Schema schema;
+  private final boolean isInternal;
+
+  String rowAvroSchemaString;
```

Comment:
never used

Reply:
Removed it. It was there from previous avro implementation.